packages/cli-utils/src/commands/turbo/thread.ts

```diff
@@ -27,18 +28,26 @@ async function* _runTurbo(params: TurboParams): AsyncIterableIterator<TurboSigna
   // NOTE: We add our override declaration here before loading all files
   // This sets `__cacheDisabled` on the turbo cache, which disables the cache temporarily
   // If we don't disable the cache then we couldn't regenerate it from inferred types
-  project.createSourceFile('__gql-tada-override__.d.ts', DECLARATION_OVERRIDE, {
-    overwrite: true,
-    scriptKind: ScriptKind.TS,
-  });
+  const overrideFile = project.createSourceFile(
+    '__gql-tada-override__.d.ts',
+    DECLARATION_OVERRIDE,
+    {
+      overwrite: true,
+      scriptKind: ScriptKind.TS,
+    }
+  );
```

Comment:
side-note: seems very odd that this has changed, since the prior wasn't reformatted by Prettier before